### ISSUE TITLE
GET /v1/weights: disclose checkpoint tensors the running alias already knows

### ISSUE THREAD

**Issue by electrobrian** [brianreborn/green-roomz#20]
## Why

OBLITERATUS (and any OpenAI-compatible research client) now **asks** a local gateway for parameters instead of assuming they are hidden. Green-Roomz already knows the GGUF path, llama.cpp `/props`, and alias identity. That should be queryable.

A loopback proof recovered a 16×8 embed/unembed pair **exactly** from `GET /v1/weights` plus completion logits. DistilGPT-2/Qwen GGUF will not dump all weights on every request; they **can** disclose path, shapes, and hashes.

Consumer: OBLITERATUS `obliteratus.openai_compat.fetch_disclosed_weights` tries, in order:

1. `GET {base}/weights`
2. `GET {base}/models/{id}/weights`

Same bearer auth as `/v1/chat/completions`. 404 is fine today; please implement rather than leave it missing.

## Contract (minimum)

`GET /v1/weights` (resolved alias from `?model=` or `Authorization` session default) and `GET /v1/models/{alias}/weights`:

```json
{
  "object": "weights",
  "model": "tool-router-agent",
  "checkpoint_path": "C:\\\\LocalAI\\\\Qwen3-0.5B.gguf",
  "format": "gguf",
  "tensors": {
    "token_embd.weight": {"shape": [151936, 1024], "dtype": "f16", "sha256": "<hex>"},
    "output.weight": {"shape": [151936, 1024], "dtype": "f16", "sha256": "<hex>"}
  }
}
```

- Default: **names, shapes, dtype, sha256, checkpoint_path**. No multi-GB bodies.
- Optional `?include=values` (or `include=values&max_params=4096`): inline nested JSON arrays **only** when the tensor is tiny. Refuse large dumps with `413` and a pointer to the file.
- Auth: same as chat (`GREEN_ROOMZ_API_KEY` / `try-local`).
- Add both paths to `EXPLICIT_ROUTES` / `GET_ONLY` in `src/gateway.mjs`.

## Out of scope

Rewriting GGUF in place. OBLITERATUS EXCISE still needs a file sidecar. This issue is **tell what you know**.

## Acceptance

- `curl -H "Authorization: Bearer try-local" http://127.0.0.1:8080/v1/weights` returns JSON for the default/resident alias.
- `GET /v1/models/tool-router-agent/weights` returns that alias even if another model is warm.
- Missing backend → `404`/`503` with the usual Green-Roomz error object, not an HTML 404.
- Node test in `test/` covering auth + shape of the payload.

Refs: llama.cpp `/props`, GGUF header, OBLITERATUS `openai_compat.introspect`.

**Comment by electrobrian** [comment]
Implemented in https://github.com/brianreborn/green-roomz/pull/24 (release/1.0.0). Live e2e covered this; fork tag https://github.com/electrobrian/green-roomz/releases/tag/v1.0.0 pending merge to tag upstream.
